phpunit.xml.dist

```diff
@@ -36,8 +30,7 @@ It's auto-generated by sonata-project/dev-kit package.
     <php>
         <ini name="precision" value="8" />
         <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0" />
-        <env name="SYMFONY_PHPUNIT_REMOVE" value="symfony/yaml" />
-        <env name="SYMFONY_PHPUNIT_VERSION" value="8.5" />
+        <env name="SYMFONY_PHPUNIT_VERSION" value="9.4" />
```

Comment:
This change was applied to dev-kit but not propagated here because the file is ignored explicitly on dev-kit: https://github.com/sonata-project/dev-kit/blob/master/config/projects.yaml#L199